packages/generator-single-spa/src/react/templates/package.json

```diff
@@ -41,6 +41,7 @@
     "single-spa-react": "^2.10.4",
     "style-loader": "^1.0.1",
     "systemjs-webpack-interop": "^1.1.2",
+    "ts-config-single-spa": "beta",
```

Comment:
I had to publish this to npm in order for the e2e tests to work. Will change this to the correct version after the merge.